### DISCUSSION TITLE
Exclude current time slice from line chart

### DISCUSSION THREAD

**Discussion by IanElsinga** [datalust/seq-tickets#1524] [Help] [answered]
For a line chart with a query like `select count(*) as count
from stream 
where EventId = 25
group by Type, time(15m)
limit 10000`, is it possible to exclude the last 15 minute data point (or whatever the time grouping happens to be)?  

Since it starts at zero and gradually fills up the 15 minute point, it makes it look as if there is an issue with the system for a few minutes. This is a problem if this represents transaction counts, for example, and if this is in a kiosk setup with non-technical viewers. 

Thanks very much.

**Comment by nblumhardt** [comment, marked as answer]
Hi Ian, thanks for the note. It's not possible to do this today, but it's something we'll keep in mind for the future (we hit this ourselves using Seq and would like to improve the experience a bit).

Best regards,
Nick
